pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "chti"
+version = "0.1.0"
+description = ""
+authors = ["Sébastien Eustace <sebastien@eustace.io>"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+plac = "^1.1"
+scikit-learn = "^0.22.2"
+jsonpickle = "^1.3"
+matplotlib = "^3.2.1"
+colorhash = "^1.0.2"
+pandas = "^1.1.2"
+numpy = "^1.19.2"
+dask = "^2.26.0"
```

Comment:
yeah the line here is:
dask = { version = "^2.9", extras = ["dataframe"]}

you just need to add the extras, like in pip. If you edit the file and then `poetry update` you should get the packages you need.